### PR TITLE
LPS-192818 

### PR DESCRIPTION
**Pull request by magjed4289**
https://issues.liferay.com/browse/LPS-192818

### CODE REVIEW

**Comment by magjed4289** [comment]
ci:test:relevant

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:test:relevant

**Comment by magjed4289** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
ci:test:stable

**Comment by magjed4289** [comment]
ci:test:relevant

**Comment by magjed4289** [comment]
ci:test:stable

File: modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIFilterRelevantObjectEntryModelListenerTest.java [hunk 1]
```diff
@@ -87,4 +111,245 @@ public void test() throws Exception {
...
+				null,
+				"headless-builder/applications" +
+					"?nestedFields=apiApplicationToAPIEndpoints," +
+						"apiEndpointToAPIFilters&nestedFieldsDepth=2",
+				Http.Method.GET
+			).toString(),
+			JSONCompareMode.LENIENT);
+	}
+
+	private void _addAPIApplication(
```
Comment (LuismiBarcos) [inline modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIFilterRelevantObjectEntryModelListenerTest.java:191]: Could we use these new methods in the existing test? So we create the application in the same way in all tests

**Review by LuismiBarcos — COMMENTED** [review]
LGTM :) 
IMHO, the tests you added are simple and easy to read, I like it 👏 👏 👏 

Just a single comment about reusing these new private methods in the existing test if it is possible

**Comment by magjed4289** [comment]
ci:test:relevant

**Comment by magjed4289** [comment]
ci:test:stable

**Comment by magjed4289** [comment]
Test failures are not related to the PR.

**Review by LuismiBarcos — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:forward:force

**Comment by magjed4289** [comment]
ci:forward:force

**Pull request closed without merging**